home/modules/ROOT/pages/index.adoc

```diff
@@ -2,15 +2,14 @@
 :navtitle: home
 :page-role: home
 
-
 ====
 * xref:3.11@tigergraph-server:release-notes:index.adoc[TigerGraph Server 3.11.0 Preview] was released on Oct 25, 2024.
 
 * xref:3.10@tigergraph-server:release-notes:index.adoc#_fixed_and_improved_in_3_10_2[TigerGraph Server 3.10.2] was released on Oct 18, 2024.
 
 * xref:4.1@tigergraph-server:release-notes:index.adoc[TigerGraph Server 4.1.0 Preview] was released on Aug. 30, 2024.
 
-* xref:cloudBeta:overview:index.adoc[TigerGraph Cloud 4.0 beta] was released on Apr. 29, 2024.
+* xref:cloud4:overview:index.adoc[TigerGraph Cloud 4] was released on Apr. 29, 2024.
```

Comment:
@samzhou2 You should not rewrite history.
TigerGraph  Cloud 4.0 Beta **WAS** published on Apr 29, 2024. It was Beta in April.

Rather than changing history, you should announcement the new release of TG Cloud 4.1 (?)
We are moving all the historical items to the bottom of the page.
Currently, we have zero announcements at the top, but I think we can have one latest announcement at the top.

Reply:
Added "beta" back, but we will need to change the URL since when we release cloud GA, beta docs will no longer available and should be removed. Added redirect rules for beta to direct to cloud4 overview page